### DISCUSSION TITLE
Create tinyhttp GH Organization

### DISCUSSION THREAD

**Discussion by shadowtime2000** [talentlessguy/tinyhttp#239] [General]
How would you feel about creating a tinyhttp gh organization? I think it would give a sense of more of an organized project and also as more repos may be created around tinyhttp it may be a good idea

**Comment by v1rtl** [comment]
I certainly thought about that many times but I'm still unsure whether I want to create a tinyhttp only gh org of something beyond this

**Comment by v1rtl** [comment]
yo, it's now in a separate org
